input/postgres/statements.go

```diff
@@ -180,7 +156,7 @@ func GetStatements(ctx context.Context, c *Collection, db *sql.DB, showtext bool
 	}
 
 	usingStatsHelper := false
-	if statementStatsHelperExists(ctx, db, showtext) {
+	if c.HelperExists("get_stat_statements", []string{"boolean"}) || (showtext && c.HelperExists("get_stat_statements", nil)) {
```

Comment:
Its worth noting that this logic is slightly different from before (its basically more precise).

This condition says that we either:
- Need the helper with arg that allows us to differentiate between with-query-text and without-query-text modes
- Or if we're looking for the version with text we can also use the old-style helper that doesn't take the argument (and always returns with text)

I think the practical intent here is that we don't fetch the version with query text every once-a-minute. Now, whether that actually works well in practice I'm not 100% sure (maybe we should just not support the old-style helper here at all anymore?), but its the same intent as before.